src/anomalib/data/ucsd_ped.py

```diff
@@ -169,8 +169,8 @@ def __init__(
         category: str,
         transform: A.Compose,
         split: Split,
-        clip_length_in_frames: int = 1,
-        frames_between_clips: int = 1,
+        clip_length_in_frames: int = 2,
+        frames_between_clips: int = 10,
```

Comment:
Can you explain this change?

Reply:
AI Vad uses these parameters so I decided to keep these as the default value for USCD. This way people can use `anomalib fit --model anomalib.models.AIVad --data anomalib.data.USCD `

Reply:
thanks, it is clear now